docs/tactics.md

```diff
@@ -336,6 +336,24 @@ refine_struct ({ .. } : semigroup α),
 { have field := @semigroup.mul_assoc, ... },
 ```
 
+### back
+
+`back hs` performs 'backwards-reasoning', repeatedly trying to apply lemmas from the list `hs` and
+the current context to the goal. `hs` can contain user attributes: in this case all theorems with 
+this attribute are added to the list of rules.
+
+Additionally, any lemmas tagged with the attribute `[back!]` are automatically included (and can be
+excluded using `-h`).
+
+`back` is successful if it applies any of the lemmas included by `hs`. Additional lemmas can be
```

Comment:
This should mention the difference between `foo` and `!foo` in `hs`.

Reply:
Done!